backend/src/api/blocks.ts

```diff
@@ -678,6 +739,62 @@ class Blocks {
   public getCurrentBlockHeight(): number {
     return this.currentBlockHeight;
   }
+
+  public async $indexCPFP(hash: string, height: number): Promise<void> {
+    let transactions;
+    if (Common.blocksSummariesIndexingEnabled()) {
```

Comment:
This code seems to not work - my `cpfp_clusters` and `transactions` tables are completely empty. Going to disable it for now.
```suggestion
    if (false/*Common.blocksSummariesIndexingEnabled()*/) {
```